src/ai/backend/manager/registry.py

```diff
@@ -510,10 +513,26 @@ async def create_session(
                     session,
                     [image_ref],
                 )
-            if (
-                _owner_id := image_row.labels.get("ai.backend.customized-image.owner")
-            ) and _owner_id != f"user:{user_scope.user_uuid}":
-                raise ImageNotFound
+
+            async with self.db.connect() as db_conn:
+                client_ctx = ClientContext(
+                    db=self.db,
+                    domain_name=user_scope.domain_name,
+                    user_id=user_scope.user_uuid,
+                    user_role=UserRole(user_scope.user_role),
+                )
+
+                perm_ctx = await get_permission_ctx(
+                    db_conn, client_ctx, SystemScope(), ImagePermission.CREATE_CONTAINER
+                )
```

Comment:
Note: To share images between teams, use SystemScope. To ensure that only the user who committed the image (the owner) can use it according to the existing logic, use UserScope.

Reply:
IMHO, It would be nice if users could set the sharing scope to either "private" or "team-wide" when committing an image.
For now, I will use `UserScope` to maintain the existing logic.